Packs/Active_Directory_Query/Scripts/IAMInitADUser/IAMInitADUser.py

```diff
@@ -0,0 +1,72 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+import traceback
+
+SUBJECT = 'Active Directory Account Created'
+DEFAULT_OUTGOING_MAPPER = "User Profile - Active Directory (Outgoing)"
+MAPPING_TYPE = "'User Profile'"
+
+
+def main():
+    try:
+        args = demisto.args()
+        pwd_generation_script = args.get("pwdGenerationScript")
+        user_profile = args.get("userProfile")
+        to_email = args.get("email")
+        mapper_in = args.get("mapperIn", DEFAULT_OUTGOING_MAPPER)
+
+        if not user_profile:
+            # no user was created
+            return
+
+        # Generate a random password
+        outputs = demisto.executeCommand(pwd_generation_script, {})
```

Comment:
fail the script in case error returned
```suggestion
        outputs = demisto.executeCommand(pwd_generation_script, {})
        if is_error(outputs):
            return_results(outputs)
```